tests/test_logging.py

```
@@ -177,5 +220,135 @@ def test_log_events(caplog, capfd):
+    ]
+
+    if not stream:
+        cmd.extend(["--logger-test-outfile", outfile.name])
+    if has_formatter:
+        cmd.append("--logger-test-has-formatter")
+    if has_filter:
+        cmd.append("--logger-test-has-filter")
+    if needs_rulegraph:
+        cmd.append("--logger-test-needs-rulegraph")
+
+    result = sp.run(
```

Ruff (0.14.6)

`subprocess` call: check for execution of untrusted input

(S603)